src/scaffoldmaker/annotation/bladder_terms.py

```diff
@@ -4,19 +4,39 @@
 
 # convention: preferred name, preferred id, followed by any other ids and alternative names
 bladder_terms = [
-    ("urinary bladder", "UBERON:0001255", "FMA:15900"),
-    ("neck of urinary bladder", "UBERON:0001258", "FMA:15912"),
-    ("Dome of the Bladder", "ILX:0738433"),
-    ("serosa of urinary bladder", "UBERON:0001260"),
     ("bladder lumen", "UBERON:0009958"),
-    ("serosa of body of urinary bladder", "ILX:0739276"),
+    ("dome of the bladder", "ILX:0738433"),
+    ("dorsal part of bladder lumen", "None"),
+    ("dorsal part of lumen of body of urinary bladder", "ILX:0739250"),
+    ("dorsal part of lumen of neck of urinary bladder", "ILX:0739255"),
+    ("dorsal part of lumen of urethra", "ILX:0739260"),
+    ("dorsal part of the bladder", "None"),
```

Comment:
Rename to remove "the" --> "dorsal part of bladder" for consistency with other terms.
Also "ventral part of bladder" below.
(Exception: "dome of the bladder" is already in SciCrunch with "the", so can't change it).

Reply:
done.

Reply:
Also update where they are used?

Reply:
Done Richard.